### PR TITLE
Graph ql 101 update

### PR DESCRIPTION
**Pull request by SimonR**
Refresh package versions and add example Playground queries

### CODE REVIEW

File: sessions/GraphQL-100/102/client/SOLUTION.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+
+## OrdersPage.js
+
+Retrieve orders array
+```
+const GET_ORDERS = gql`
+  query {
+    getOrders {
+	    id, orderDate, product {name}
```
Comment (tvking) [inline sessions/GraphQL-100/102/client/SOLUTION.md:52]: Testing this out the orders list seem to be populated like this: `null` `product`, `null`, `product`
Which results in this error.
image
When `Buying` a product the Id returned is also undefined. So perhaps related. 
image

File: sessions/GraphQL-100/102/client/src/components/OrdersPage.js [hunk 2]
```diff
@@ -17,7 +17,7 @@ const OrdersPage = props => (
     <Query query={GET_ORDERS}>
       {({ loading, error, data }) => {
         if (loading) return <p>Loading...</p>;
-        if (error) return <p>Oops something went wrong 😵</p>;
+        if (error) return <p>Oops something went wrong</p>;
         console.log(data.getOrders);
```
Comment (tvking) [inline sessions/GraphQL-100/102/client/src/components/OrdersPage.js:21]: Is this meant to be here?

File: sessions/GraphQL-100/102/client/src/components/OrdersPage.js [hunk 3]
```diff
@@ -17,7 +17,7 @@ const OrdersPage = props => (
...
         return data.getOrders.map(order => (
```
Comment (tvking) [inline sessions/GraphQL-100/102/client/src/components/OrdersPage.js:22]: This bit wasn't working for me locally I got it to work with 
```suggestion
        return data.getOrders.map(order => {
          if (!order) return <p>Oops something went wrong</p>;
          return <div key={order.id} className="orderRow">
            <p>Order ID: {order.id}</p>
            <p>{moment(parseInt(order.orderDate)).format("HH:mm D/M/YY")}</p>
            <p>Product: {order.product.name}</p>
          </div>
        });
```

File: sessions/GraphQL-100/102/client/README.md [hunk 4]
```diff
@@ -4,6 +4,10 @@ The following tasks give you exposure to working with graphql on the client side
...
 
-1. Fetch all products from the graphql server (components/ProductList.js:19)
-2. Fetch a product from the graphql server (components/ProductPage:19)
-3. Fetch all orders from the grapqhl server (component/OrdersPage:10)
+1. Fetch all products from the graphql server (components/ProductList.js)
+2. Fetch a product from the graphql server (components/ProductPage.js)
+3. Create a new order (components/ProductPage.js)
+4. Fetch all orders from the grapqhl server (component/OrdersPage.js)
+
+
```
Comment (tvking) [inline sessions/GraphQL-100/102/client/README.md:12]: Should this have instructions for running the `npm server` and the build and run of the client?

File: sessions/GraphQL-100/101/1-GraphQL/README.md [hunk 5]
```diff
@@ -28,4 +28,16 @@ More info: [GraphQL Tag](https://github.com/apollographql/graphql-tag)
 
 ## GraphQL Playground
 
-<TODO>
+Start the server using 
+
+```
+npm install
```
[the remark this answers is not in the document]
Comment (SimonR) [reply sessions/GraphQL-100/101/1-GraphQL/README.md:34]: Updated to the latest version of NPM and it behaves the same as your build (a-z ordering). Perhaps I could remove the yarn.lock entirely leaving just the package.json

[the remark this answers is not in the document]
Comment (tvking) [reply sessions/GraphQL-100/101/1-GraphQL/README.md:34]: yeah, do we need both in for a 101 repo?

File: sessions/GraphQL-100/101/2-GraphQL/README.md [hunk 6]
```diff
@@ -26,9 +26,68 @@ in server.js
 
 `const server = new ApolloServer({ typeDefs, resolvers });`
 
+
+## Query the server
+
+Use the GraphQL Playground console to query the server
+
```
[the remark this answers is not in the document]
Comment (SimonR) [reply sessions/GraphQL-100/101/2-GraphQL/README.md:33]: I think this is obvious enough from part 1

File: sessions/GraphQL-100/101/1-GraphQL/README.md [hunk 7]
```diff
@@ -28,4 +28,16 @@ More info: [GraphQL Tag](https://github.com/apollographql/graphql-tag)
...
+npm start
```
[the remark this answers is not in the document]
Comment (SimonR) [reply sessions/GraphQL-100/101/1-GraphQL/README.md:34]: good shout, updated

File: sessions/GraphQL-100/101/2-GraphQL/README.md [hunk 8]
```diff
@@ -26,9 +26,68 @@ in server.js
...
+        "id": "3",
+        "name": "Matt"
+      }
+    ]
+  }
+}
+```
+
+
+Return an instace of a user
```
[the remark this answers is not in the document]
Comment (SimonR) [reply sessions/GraphQL-100/101/2-GraphQL/README.md]: corrected

File: sessions/GraphQL-100/101/5-GraphQL/README.md [hunk 9]
```diff
@@ -51,3 +53,42 @@ In the Mutations we want to listen to changes for, add:
...
+Subscribe to `userCreated`
+```
+subscription {
+  userCreated {
+    id, name, image
+  }
+}
+```
+
+Add new user (new query)
```
[the remark this answers is not in the document]
Comment (SimonR) [reply sessions/GraphQL-100/101/5-GraphQL/README.md]: good shout, I've added the tab guidance

File: sessions/GraphQL-100/102/client/SOLUTION.md (hunk 1 above)
Comment (SimonR) [reply sessions/GraphQL-100/102/client/SOLUTION.md:52]: added missing `id` into solutions.md

File: sessions/GraphQL-100/102/client/src/components/OrdersPage.js (hunk 2 above)
Comment (SimonR) [reply sessions/GraphQL-100/102/client/src/components/OrdersPage.js:21]: I removed the emoji to resolve a warning message

File: sessions/GraphQL-100/102/client/src/components/OrdersPage.js (hunk 3 above)
Comment (SimonR) [reply sessions/GraphQL-100/102/client/src/components/OrdersPage.js:22]: It works fine for me, your suggestion didn't work?

**Pull request merged**